### ISSUE TITLE
Andriod - Will there ever be support for front ends like dig?

### ISSUE THREAD

**Issue by SamP99** [citra-emu/citra-android#195]
Will front ends every be supported, currently dig front end only launches the citra app itself and not the selected game.

Dig needs a "file path key".

Does this already exist or is it yet to be implemented

**Comment by Retroism** [comment]
UP! to this
Dolphin already does this and hopefully Citra android too in future releases. Using MainActivity only opens the app but doesn't launch it from a front end. It either needs a file path key or it still doesn't support it.

**Comment by bigguynak** [comment]
I agree with this.  A new issue was opened for this same thing.  Issue #222
